cpp/src/barretenberg/serialize/msgpack_impl/concepts.hpp

```diff
@@ -4,12 +4,10 @@ struct DoNothing {
     void operator()(auto...) {}
 };
 namespace msgpack_concepts {
-template <typename T> concept HasMsgPack = requires(T t, DoNothing nop)
-{
-    t.msgpack(nop);
-};
-template <typename T> concept HasMsgPackPack = requires(T t, DoNothing nop)
-{
-    t.msgpack_pack(nop);
-};
+template <typename T>
```

Comment:
nit: can a newline go between each of these concept definitions, this took my egg brain a while to parse